llama_index/llms/openai_like.py

```diff
@@ -58,10 +71,19 @@ def _tokenizer(self) -> Tokenizer:
                     "huggingface tokenizers with OpenAILike."
                 ) from exc
 
-            return AutoTokenizer.from_pretrained(self._get_model_name())
-        else:
-            return self.tokenizer
+            return AutoTokenizer.from_pretrained(self.tokenizer)
+        return self.tokenizer
 
     @classmethod
     def class_name(cls) -> str:
         return "OpenAILike"
+
+
+# Use these as kwargs for OpenAILike to connect to LocalAIs
+localai_defaults: Mapping[str, Any] = MappingProxyType(
```

Comment:
should we maybe add a notebook to show how to do this?

Reply:
Yeah I can make a Notebook, wanted to get team onboard before I invested the time in a demo Notebook